admin/resources/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -50,13 +62,110 @@ type FormValues = Pick<
   | "expectedSalary"
   | "locationPreferences"
   | "status"
-> & {
-  acceptedOperationalRequirements: string[] | undefined;
-  cmoAssets: string[] | undefined;
-  expectedClassifications: string[] | undefined;
-  pool: string;
-  user: string;
+> &
+  Pick<
+    User,
+    "email" | "firstName" | "lastName" | "preferredLang" | "telephone"
+  > & {
+    acceptedOperationalRequirements: string[] | undefined;
+    cmoAssets: string[] | undefined;
+    expectedClassifications: string[] | undefined;
+    pool: string;
+    user: string;
+    userMode: UserModeValue;
+  };
+
+type UserModeValue = "existing" | "new";
+
+const UserFormSection: React.FunctionComponent<{
+  control: Control<FormValues>;
+  userOptions: Option<string>[];
+}> = ({ control, userOptions }) => {
+  const userMode = useWatch({
+    control,
+    name: "userMode",
+  });
+  const intl = useIntl();
+  return (
+    <>
+      <div
+        data-h2-visibility={userMode === "existing" ? undefined : "b(hidden)"}
+      >
+        <Select
+          id="user"
+          label={intl.formatMessage(poolCandidateMessages.userLabel)}
+          nullSelection={intl.formatMessage(
+            poolCandidateMessages.userPlaceholder,
+          )}
+          name="user"
+          options={userOptions}
+          rules={{
+            required:
+              userMode === "existing" ? errorMessages.required : undefined,
+          }}
+        />
+      </div>
+      <div data-h2-visibility={userMode === "new" ? null : "b(hidden)"}>
+        <Input
```

Comment:
So hydrogen's parsing engine doesn't recognize expressions like this. You'll have to define the expression above in a format that hydrogen can pick up, and then pass it down after. 
![image](https://user-images.githubusercontent.com/22059495/143064555-1c48099c-25cb-4602-936f-e61ef167f089.png)

Reply:
Good catch!  Thanks.  Also, I've added the sections like the update page.